backend/src/main/java/gov/cdc/usds/simplereport/config/AzureTestEventReportingQueueConfiguration.java

```diff
@@ -53,12 +55,23 @@ public CompletableFuture<Void> reportAsync(TestEvent testEvent) {
       log.warn(
           "No TestEventReportingService configured; defaulting to no-op reporting for TestEvent [{}]",
           testEvent.getInternalId());
+      String buffer = toBuffer(testEvent);
+      log.info("TestEvent serializes as: {}", buffer);
       return CompletableFuture.completedFuture(null);
     }
 
     @Override
     public void markTestEventsAsReported(Set<TestEvent> testEvents) {
       log.warn("No TestEventReportingService configured; defaulting to no-op reporting");
     }
+
+    private String toBuffer(TestEvent testEvent) {
+      try {
```

Comment:
These changes are not strictly required for this PR, but they do make it easier to catch test serialization issues locally. Happy to revert if reviewers prefer though.

Reply:
looks good, would a `toJson` be a better name?

Reply:
I'd personally vote to keep it as `toBuffer` since that's the name in the prod version (https://github.com/CDCgov/prime-simplereport/blob/cdc41c6b31f37292697029b3ad6820ecd5c96294/backend/src/main/java/gov/cdc/usds/simplereport/service/AzureStorageQueueTestEventReportingService.java#L79-L80) but don't have terribly strong feelings about it.